src/SFA.DAS.EmployerApprenticeshipsService.Domain/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerCommitments.Domain.Extensions
+{
+    public static class StringExtensions
+    {
+        public static T ToEnum<T>(this string value) where T : struct
+        {
+            return (T) Enum.Parse(typeof(T), value);
```

Comment:
I would personally use Enum.TryParse() to avoid exceptions but this should be fine too

Reply:
I think in this case I need to throw an exception, but you're suggestion can mean the exception can be a custom exception